integration/specs/TimePicker/timePicker-1.spec.js

```diff
@@ -0,0 +1,576 @@
+        const picklist = new PageTimePicker(TIME_PICKER);
+        picklist.click();
+        picklist.waitUntilOpen();
+        picklist.setMinutesValue('56');
+        browser.keys(ARROW_LEFT_KEY);
+        browser.keys(DELETE_KEY);
+        expect(picklist.getMinutesValue()).toBe('');
+    });
+    it('should set hour input value to "01" when press up key after reset hour input', () => {
+        const picklist = new PageTimePicker(TIME_PICKER);
+        picklist.click();
+        picklist.waitUntilOpen();
+        picklist.setHourValue('05');
+        browser.keys(ARROW_LEFT_KEY);
+        browser.keys(DELETE_KEY);
+        browser.keys(ARROW_UP_KEY);
+        expect(picklist.getHourValue()).toBe('01');
+    });
+    it('should set hour input value to "01" when press up button after reset hour input', () => {
+        const picklist = new PageTimePicker(TIME_PICKER);
+        picklist.click();
+        picklist.waitUntilOpen();
+        picklist.setHourValue('05');
+        browser.keys(ARROW_LEFT_KEY);
+        browser.keys(DELETE_KEY);
+        picklist.clickUpButton();
+        expect(picklist.getHourValue()).toBe('01');
+    });
+    it('should set hour input value to "12" when press down key after reset hour input', () => {
+        const picklist = new PageTimePicker(TIME_PICKER);
+        picklist.click();
+        picklist.waitUntilOpen();
+        picklist.setHourValue('05');
+        browser.keys(ARROW_LEFT_KEY);
+        browser.keys(DELETE_KEY);
+        browser.keys(ARROW_DOWN_KEY);
+        expect(picklist.getHourValue()).toBe('12');
+    });
+    it('should set hour input value to "12" when press down button after reset hour input', () => {
+        const picklist = new PageTimePicker(TIME_PICKER);
+        picklist.click();
+        picklist.waitUntilOpen();
+        picklist.setHourValue('05');
+        browser.keys(ARROW_LEFT_KEY);
+        browser.keys(DELETE_KEY);
+        picklist.clickDownButton();
+        expect(picklist.getHourValue()).toBe('12');
+    });
+    it('should set minutes input value to "59" when press down key after reset minutes input', () => {
+        const picklist = new PageTimePicker(TIME_PICKER);
+        picklist.click();
+        picklist.waitUntilOpen();
+        picklist.setMinutesValue('34');
+        browser.keys(ARROW_LEFT_KEY);
+        browser.keys(DELETE_KEY);
+        browser.keys(ARROW_DOWN_KEY);
+        expect(picklist.getMinutesValue()).toBe('59');
+    });
+    it('should set minutes input value to "59" when press down button after reset minutes input', () => {
+        const picklist = new PageTimePicker(TIME_PICKER);
+        picklist.click();
+        picklist.waitUntilOpen();
+        picklist.setMinutesValue('34');
+        browser.keys(ARROW_LEFT_KEY);
+        browser.keys(DELETE_KEY);
+        picklist.clickDownButton();
+        expect(picklist.getMinutesValue()).toBe('59');
+    });
```

Comment:
remove this or better if you want combine them ones with keys and others with button